doc/20_Extending_Pimcore/13_Bundle_Developers_Guide/06_Event_Listener_UI.md

```diff
@@ -29,26 +29,16 @@ class PimcoreAdminListener
 {
     public function addCSSFiles(PathsEvent $event): void
     {
-        $event->setPaths(
-            array_merge(
-                $event->getPaths(),
-                [
-                    '/admin-static/css/admin-style.css'
-                ]
-            )
-        );
+        $event->addPaths([
+            '/admin-static/css/admin-style.css',
+        ]);
     }
 
     public function addJSFiles(PathsEvent $event): void
     {
-        $event->setPaths(
-            array_merge(
-                $event->getPaths(),
-                [
-                    '/admin-static/js/startup.js'
-                ]
-            )
-        );
+        $event->addPaths([
+            '/admin-static/js/startup.js',
+        ]);
     }
```

Comment:
Thank you for your PR!


Just found some other instances 
https://github.com/pimcore/pimcore/blob/82b7f2d53448694aa6d1e42d5c3e92dfa8722117/doc/26_Best_Practice/71_Style_Backend_Depending_On_App-Env.md?plain=1#L45
https://github.com/pimcore/pimcore/blob/8e119b9e110189e94af4366697b32317495f4a09/doc/20_Extending_Pimcore/13_Bundle_Developers_Guide/09_Adding_Document_Editables.md?plain=1#L77-L78

Could you also please apply the changes in this PR?

Reply:
@kingjia90 Done https://github.com/pimcore/pimcore/pull/17411/commits/1828e4f5493172ad270a63aebed1be81e16c8db6